packages/utils/src/lib/file-system.unit.test.ts

```diff
@@ -139,3 +140,11 @@ describe('findLineNumberInText', () => {
     expect(findLineNumberInText(``, 'x')).toBeNull();
   });
 });
+
+describe('filePathToCliArg', () => {
```

Comment:
❔ Does this make the path "invalid" in face of `node:path`? If so, do I understand it correctly that this has to be the last operation done on the path (i.e. after path separator unification)?

Reply:
It's up to whoever created the path to apply proper path separator. This function is only concerned with what needs to be handled when it's used as a command-line argument.